.travis.yml

```diff
@@ -40,7 +40,7 @@ jobs:
   include:
     - stage: sentinel
       # All other jobs will be cancelled if the sentinel job fails
-      python: 3.6
+      python: 3.6.9
```

Comment:
Why is this change required? It should work fine with any version of 3.6

Reply:
tests module was failing on travis.ci with python version 3.6, you can have a look at [this](https://travis-ci.com/github/anshalshukla/coala/jobs/490860929). But it was working fine on my local machine with python3.6.9 and pip9.0.3 so I tried running the build using that configuration which worked (you can look at [this](https://travis-ci.com/github/anshalshukla/coala/jobs/490861059).

Reply:
Yeah, strange behavior. Might need to look a bit more in this. For now, this is fine.